docs/development.rst

```diff
@@ -388,6 +388,14 @@ recommended process for generating a new release is described here.
 
         python setup.py sdist upload
 
+10. Ensure that a built Docker image is tagged on
+    `Girder's Docker Hub page <https://hub.docker.com/r/girder/girder/tags/>`_.
+    Such a build can be added by following the instructions for
+    `adding and running a new build <https://docs.docker.com/docker-hub/builds/#add-and-run-a-new-build>`_.
```

Comment:
Docker hub automatically builds master.  If you pull the docker image before any other commits are merged to master, you can just tag it with the release information.  This ensures that the built image is as clean as docker hub makes it.